src/spread/obj-symbol-property.case

```diff
@@ -0,0 +1,33 @@
+// Copyright (C) 2017 Caio Lima. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+desc: Spread operation where source object contains Symbol properties
+template: default
+esid: pending
+info: |
+    Pending Runtime Semantics: PropertyDefinitionEvaluation
+
+    PropertyDefinition:...AssignmentExpression
+
+    1. Let exprValue be the result of evaluating AssignmentExpression.
+    2. Let fromValue be GetValue(exprValue).
+    3. ReturnIfAbrupt(fromValue).
+    4. Let excludedNames be a new empty List.
+    5. Return CopyDataProperties(object, fromValue, excludedNames).
+
+---*/
+
+//- setup
+let o = {};
+o[Symbol('foo')] = 1;
+
+//- args
+{...o, c: 4, d: 5}
+//- params
+obj
+//- body
+
+assert.sameValue(obj[Symbol('foo')], 1);
```

Comment:
this will fail. You should store this symbol in a variable or use Symbol.for. Everytime you call Symbol('foo') it creates a new symbol. Storing a symbol will preserve it or using Symbol.for will find that reference.